# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/tests/reftest/1961387-2-ref.html
# Commit: ea30c0eca521
# Full Hash: ea30c0eca521cf3ffc2fcc60a3b5cd547d8b0500
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2025-05-07 04:19:47
# Regressor Bug: 1961387
# File Overlap Count: 3
# Description:
#   Bug 1961387 - Add a reftest with larger glyphs, for CBDT version of Noto Color Emoji. r=gfx-reviewers,lsalzman
#   
#   Without the scaling fix, the advance of the bitmap emoji glyph isn't
#   scaled according to the rendered size, and so the large glyph extends
#   beyond the expected rect.
# ==============================================================================

diff -r 057a31f469ac -r ea30c0eca521 gfx/tests/reftest/1961387-2-ref.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/tests/reftest/1961387-2-ref.html	Tue May 06 20:12:46 2025 +0000
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<style>
+    div {
+        position: absolute;
+        writing-mode: vertical-lr;
+        font: 250px serif, Noto Color Emoji;
+    }
+    #mask {
+        color: transparent;
+        background: green;
+    }
+</style>
+
+<p>The emoji glyph should not be visible outside the green box:</p>
+
+<div id="mask">.&#x1f400;.</div>